qingstor/sdk/config.py

```diff
@@ -96,8 +105,15 @@ def load_default_config(self):
         self.load_config_from_data(config_data)
         return self
 
+    def load_from_env(self, key):
+        key = f"QINGSTOR_{key.upper()}"
+        v = os.environ.get(key)
+        return v
```

Comment:
```suggestion
        return os.environ.get(key)
```

Reply:
BTW, we should not load `host`, `port` or `protocol` from env as them will be remove in later releases.

Reply:
I think it's better not to remove them, but to keep forward compatibility